src/foam/u2/dialog/ApplicationPopup.js

```diff
@@ -146,6 +151,13 @@ foam.CLASS({
     {
       class: 'Array',
       name: 'primaryActions'
+    },
+    {
+      class: 'foam.u2.ViewSpec',
+      name: 'progressView',
+      value: {
```

Comment:
Doesn't just value: 'foam.u2.wizard.views.ProgressBarView' work?

Reply:
Not in this case since we add extra props to when we render this